packages/gatsby-admin/src/pages/index.tsx

```diff
@@ -1,5 +1,72 @@
 import React from "react"
-import { useQuery } from "urql"
+import { useQuery, useMutation } from "urql"
+
+const InstallInput = () => {
+  const [value, setValue] = React.useState("")
+
+  const [_, installGatbyPlugin] = useMutation(`
+    mutation installGatsbyPlugin($name: String!) {
+      createNpmPackage(npmPackage: {
+        name: $name,
+        dependencyType: "development"
```

Comment:
Just for my own curiosity: What does `dependencyType: "development"` mean in this context?

Reply:
The npm package gets added to the `devDependencies`!

Reply:
Why tho? You typically add your plugin's npm package to `dependencies` 🤔

Reply:
To me, anything that isn't shipped to the browser is a devDep—I don't really care either way though, we can bikeshed that 😬🤷‍♂️

Reply:
Our docs use `dependencies` so let’s roll with that 😀